README.md

```diff
@@ -2,14 +2,14 @@
 
 Barcelona on Rails is dedicated to providing a harassment-free user group experience for everyone, regardless of sex, sexual orientation, gender identity, disability, physical appearance, body size, race, religion or nationality. We do not tolerate harassment of participants in any form.
 
-Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
+Harassment includes offensive verbal comments related to sex, sexual orientation, gender identity, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
 
-Sexual language and imagery is not appropriate for any meeting or user group venue, including talks. Be careful in the words that you choose. Do not insult or put down other attendees. Behave civilly. Remember that sexist, racist, and other exclusionary jokes can be offensive to those around you. Offensive statements or jokes are not appropriate for the meetings of our user group.
+Sexual language and imagery is not appropriate for any meeting or user group venue, including talks. Be careful in the words that you choose. Do not insult or put down other attendees. Behave civilly. Remember that sexist, racist, and other exclusionary jokes can be offensive to those around you. Offensive statements or jokes are not appropriate for the meetings and discussions of our user group.
 
 Be kind to others.
 
 Participants asked to stop any harassing behavior are expected to comply immediately.
-Anyone violating these rules repeatedly or unapologetically will be asked to leave the meeting, and will be refused entrance to future meetings.
+Anyone violating these rules repeatedly or unapologetically will be asked to leave the meeting, and may be refused entrance to future meetings.
```

Comment:
I very much like that. 😃 

I was not sure how to soften the wording - but I guess that was because I was thinking around the `asked to leave` part. I am a fan of the concept of forgiveness.